tests/rspec/spec/existing_cluster_spec.rb

```
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'shared_examples/k8s'
+require 'env_var'
+
+RSpec.describe 'existing_cluster' do
+  before(:all) do
+    raise 'Missing existing platform. Please set PLATFORM environment variable' unless EnvVar.set?(%w[PLATFORM])
```

Great, I like the "fail fast" approach!
Can you do the same with the `CLUSTER` variable?

---

done